modules/extension/xsd/xsd-fes/src/main/java/org/geotools/filter/v2_0/bindings/BBOXTypeBinding.java

```diff
@@ -52,4 +66,48 @@
     public QName getTarget() {
         return FES.BBOXType;
     }
+
+    public Class getType() {
+        return BBOX.class;
+    }
+
+    @Override
+    public Object getProperty(Object object, QName name) throws Exception {
+        BBOX box = (BBOX) object;
+        
+        if (FES.ValueReference.equals(name)) {
+            if (box.getExpression1() instanceof PropertyName) {
```

Comment:
I thought the assumption of property name was finally being lifted in filter 2.0? Let me check - seems to be handled by a value reference now.

Probably something to look at later...